src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/totrinnskontroll/SendTilBeslutterSteg.kt

```diff
@@ -93,6 +93,7 @@ class SendTilBeslutterSteg(
     private fun validerRiktigTilstandVedInvilgelse(saksbehandling: Saksbehandling) {
         val vedtaksresultat = vedtaksresultatService.hentVedtaksresultat(saksbehandling)
         if (vedtaksresultat == TypeVedtak.INNVILGELSE) {
+            // TODO: SJekk
             brukerfeilHvisIkke(vilkårService.erAlleVilkårOppfylt(saksbehandling.id)) {
                 "Kan ikke innvilge hvis ikke alle vilkår er oppfylt for behandlingId: ${saksbehandling.id}"
```

Comment:
Dobbel negasjon er aldri helt enkelt. Hva med 
```suggestion
                "Alle vilkår må være oppfylt før du kan innvilge."
```
eller
```suggestion
                "Det finnes vilkår som ikke er oppfylt. Alle vilkår må være oppfylt før du kan innvilge."
```

Reply:
Fikset i egen pr